docs/content/sdk/_index.md

```diff
@@ -10,8 +10,52 @@ bookSearchExclude: false
 
 # SDKs
 
-This is the primary section for Ensign users.
+Welcome Ensign user!
+
+If you're looking to figure out how to write code to connect to Ensign, you've come to the right place.
 
 <!--more-->
 
-<!-- replace this comment with the content of the article -->
\ No newline at end of file
+Ensign is written primarily in Go and Protocol Buffers because that's what we like, but hey, this isn't about us, it's about **you**.
+
+We want Ensign to be as accessible as possible, so we made you some SDKs 💙
+
+## Available SDKs
+
+Here is a list of the SDKs that are currently available.
+
+You should be able to install each using your usual language-specific package manager (e.g. `go get`, `pip install`, `npm install`, etc.):
+
+- [go-ensign](https://github.com/rotationalio/go-ensign)
+- [pyensign](https://github.com/rotationalio/pyensign)
+- [ensignJS](https://github.com/rotationalio/ensignjs) *Note: 🙈 this just an empty repo for now, but we're working on it!!*
+
+Each SDK is structured in a language-specific fashion and contains cross-sdk driver methods as well as tooling that may be tailored to users of specific languages.
+
+Don't see an Ensign SDK for the language you love most? Tell us (support@rotational.io), and we'll get right on it!
+
+## What's in the SDK Repos?
+
+The SDK repos above contain driver and library code for interacting with Ensign. They each have their own language-specific docs.
+
+### Common Methods
+
+Here are some of the common driver methods that all of the SDKs have in some shape or form (*there are some implementation differences due to the ways that different language handle concurrency*).
+
+- Create an open `Client` to Ensign using your [API keys]({{< ref "/getting-started#getting-started" >}})
+- Create a new `Topic` on your `Client`
+- Structure valid Ensign `Events` of various possible `MimeTypes`
+- Create a new `Publisher` on your `Client` and invoke this `Publisher`'s `Publish` method to publish an `Event` to a `Topic`
+- Create a new `Subscriber` on your `Client` and create an event stream to collect published `Events` using this `Subscriber`'s `Subscribe` method
+
+## What are these Protobuf Things?
```

Comment:
Since this is the first time `Protobuf` is used in this doc, should `Protobuf` be replaced with `Protocol Buffers` here or maybe add `(protobufs)` after `Protocol Buffers` on line 19?